src/widgets/Grabber.h

```diff
@@ -46,15 +46,13 @@ class GrabberEvent:public wxCommandEvent
    GrabberEvent(wxEventType type = wxEVT_NULL,
                 wxWindowID winid = 0,
                 const wxPoint& pt = wxDefaultPosition)
-   : wxCommandEvent(type, winid)
+   : wxCommandEvent(type, winid), mPos(pt)
```

Comment:
I'll give it a read in a bit, but here's one using an object:

```
class morecomplex
{
public:
   morecomplex()
   {
   }

   morecomplex(int parm)
   {
      mParm = parm;
   }
public:
   int mParm;
};

class inconstructor
{
public:
   inconstructor(morecomplex parm1)
   {
      mParm = parm1;
   }

public:
   morecomplex mParm;
};

class ininitializer
{
public:
   ininitializer(morecomplex parm1)
   :  mParm(parm1)
   {
   }

public:
   morecomplex mParm;
};

extern morecomplex trycc;
extern morecomplex tryci;
inconstructor tryc(trycc);
ininitializer tryi(tryci);
```

And the results:

```
        .file   "t.cpp"
        .section        .text.startup,"x"
        .p2align 4,,15
        .def    _GLOBAL__sub_I_tryc;    .scl    3;      .type   32;     .endef
        .seh_proc       _GLOBAL__sub_I_tryc
_GLOBAL__sub_I_tryc:
.LFB13:
        .seh_endprologue
        movq    .refptr.trycc(%rip), %rax
        movl    (%rax), %eax
        movl    %eax, tryc(%rip)
        movq    .refptr.tryci(%rip), %rax
        movl    (%rax), %eax
        movl    %eax, tryi(%rip)
        ret
        .seh_endproc
        .section        .ctors,"w"
        .align 8
        .quad   _GLOBAL__sub_I_tryc
        .globl  tryi
        .bss
        .align 4
tryi:
        .space 4
        .globl  tryc
        .align 4
tryc:
        .space 4
        .ident  "GCC: (GNU) 4.9.2"
        .section        .rdata$.refptr.tryci, "dr"
        .globl  .refptr.tryci
        .linkonce       discard
.refptr.tryci:
        .quad   tryci
        .section        .rdata$.refptr.trycc, "dr"
        .globl  .refptr.trycc
        .linkonce       discard
.refptr.trycc:
        .quad   trycc
```

Again, I haven't yet read the article at the provided link yet, but seems to me that, so far, they are the same.